packages/ui/src/components/va-image/VaImage.vue

```diff
@@ -42,6 +43,7 @@ export default defineComponent({
     ratio: { type: Number as PropType<number>, default: 1 },
     contain: { type: Boolean as PropType<boolean>, default: false },
     src: { type: String as PropType<string>, required: true },
+    alt: { type: String, default: '' },
```

Comment:
Maybe we should make it required?

Reply:
`required` is a critically important for component functionality props, but its not that case. If user doesn't want to waste time on this, we shouldn't force him.